module/Application/config/module.config.php

```diff
@@ -67,4 +112,22 @@
             __DIR__ . '/../view',
         ],
     ],
-];
+
+    'doctrine' => [
+        'driver' => [
+            'application_entity' => [
+                'class' => AnnotationDriver::class,
+                'cache' => 'array',
+                'paths' => [
+                    __DIR__ . '/src/Application/Entity'
```

Comment:
What does `./vendor/bin/doctrine-module orm:info` report?

What is the realpath here?